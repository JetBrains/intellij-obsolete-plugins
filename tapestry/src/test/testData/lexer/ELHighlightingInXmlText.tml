<html xmlns:t="http://tapestry.apache.org/schema/tapestry_5_1_0.xsd">
    <head title="!!">
    </head>
    <body>
        <a href="#" t:type="pagelink" page="startpage">my link ${rrrrr?.tt(4).r(77, -5.3, [-3,  -2]).yy}</a>
        <p>${message:question-to-life}</p>
    </body>
</html>
