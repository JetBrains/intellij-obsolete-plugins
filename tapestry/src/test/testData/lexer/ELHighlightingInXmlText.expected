XML_START_TAG_START ('<')
XML_TAG_NAME ('html')
TAG_WHITE_SPACE (' ')
XML_NAME ('xmlns:t')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('http://tapestry.apache.org/schema/tapestry_5_1_0.xsd')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('head')
TAG_WHITE_SPACE (' ')
XML_NAME ('title')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('!!')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_END_TAG_START ('</')
XML_TAG_NAME ('head')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
        ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('a')
TAG_WHITE_SPACE (' ')
XML_NAME ('href')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('#')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
TAG_WHITE_SPACE (' ')
XML_NAME ('t:type')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('pagelink')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
TAG_WHITE_SPACE (' ')
XML_NAME ('page')
XML_EQ ('=')
XML_ATTRIBUTE_VALUE_START_DELIMITER ('"')
XML_ATTRIBUTE_VALUE_TOKEN ('startpage')
XML_ATTRIBUTE_VALUE_END_DELIMITER ('"')
XML_TAG_END ('>')
XML_DATA_CHARACTERS ('my')
XML_WHITE_SPACE (' ')
XML_DATA_CHARACTERS ('link')
XML_WHITE_SPACE (' ')
${ ('${')
TAP5_EL_IDENTIFIER ('rrrrr')
?. ('?.')
TAP5_EL_IDENTIFIER ('tt')
( ('(')
TAP5_EL_INTEGER ('4')
) (')')
. ('.')
TAP5_EL_IDENTIFIER ('r')
( ('(')
TAP5_EL_INTEGER ('77')
, (',')
WHITE_SPACE (' ')
TAP5_EL_DECIMAL ('-5.3')
, (',')
WHITE_SPACE (' ')
[ ('[')
TAP5_EL_INTEGER ('-3')
, (',')
WHITE_SPACE ('  ')
TAP5_EL_INTEGER ('-2')
] (']')
) (')')
. ('.')
TAP5_EL_IDENTIFIER ('yy')
} ('}')
XML_END_TAG_START ('</')
XML_TAG_NAME ('a')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
        ')
XML_START_TAG_START ('<')
XML_TAG_NAME ('p')
XML_TAG_END ('>')
${ ('${')
TAP5_EL_IDENTIFIER ('message')
: (':')
TAP5_EL_IDENTIFIER ('question-to-life')
} ('}')
XML_END_TAG_START ('</')
XML_TAG_NAME ('p')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
    ')
XML_END_TAG_START ('</')
XML_TAG_NAME ('body')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
XML_END_TAG_START ('</')
XML_TAG_NAME ('html')
XML_TAG_END ('>')
XML_WHITE_SPACE ('
')
